cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -195,24 +255,43 @@ subroutine alloc_grid
          yyav     (nx_block,ny_block,max_blocks), & ! mean T-cell value of yy
          hm       (nx_block,ny_block,max_blocks), & ! land/boundary mask, thickness (T-cell)
          bm       (nx_block,ny_block,max_blocks), & ! task/block id
-         uvm      (nx_block,ny_block,max_blocks), & ! land/boundary mask, velocity (U-cell)
+         uvm      (nx_block,ny_block,max_blocks), & ! land/boundary mask, velocity (U-cell) - water in case of all water point
```

Comment:
make the comment for uvm the same form as npm, epm below

Reply:
PR coming